lexer_test.go

```diff
@@ -35,6 +36,9 @@ Sleep 2`
 		{SET, "Set"},
 		{PADDING, "Padding"},
 		{NUMBER, "5"},
+		{SET, "Set"},
+		{CURSOR_BLINK, "CursorBlink"},
+		{STRING, "false"},
```

Comment:
I think making it as STRING and parsing as bool later is fine, but if you think we need another type for bool (I believe this is the first one) let me know

Reply:
Having a boolean type would be great for long term if possible.

Reply:
The bool type would also make parser errors much nicer.

We can leverage the parser errors stuff we already have:

```
Set CursorBlink fals
                ^^^^ expected boolean (false / true)
```

Reply:
Done, also updated parser to check for valid values

Reply:
Sweet, this looks fantastic!